backend/src/activitypub/objects/index.ts

```diff
@@ -129,16 +130,19 @@ export async function cacheObject(
 
 	{
 		const properties = JSON.parse(row.properties)
+
+		// prettier-ignore
```

Comment:
It's unnecessary to set `const properties = JSON.parse(row.properties)` because `row.properties === sanitizedProperties`